source/scpp/extra/DVMChecks.cpp

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+
+    Contains checking the offset of virtual methods.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+#include "DVMChecks.h"
+
+int checkVMOffset (const char* classname, const char* offsets)
+{
+    return 0;
```

Comment:
What is the purpose of this function? It just returns 0.

Reply:
@Geod24 gave me this comment.
https://github.com/bpfkorea/agora/pull/711#pullrequestreview-386915944

I just want to keep that pattern at unittest.
And I thought that A would be automatically generated in CI and the test would proceed.
I couldn't implement overloaded methods with C++ source.
So I had no choice but to make this decision.